### ISSUE TITLE
textarea: md-maxlength value char count not updated inside html

### ISSUE THREAD

**Issue by TemitaTom** [angular/material#11349]
## Bug

CodePen
https://codepen.io/anon/pen/MXZEXY?editors=1010

#### Detailed Reproduction Steps: 
create textarea with: 

md-maxlength="model.myValue.isDisabled ? '' : 500" --> WORKS
md-maxlength="model.myValue.isDisabled || model.global.isDisabled ? '' : 500" --> NOT works

EDIT: the problem is related with multiple levels of ng-include.

### What is the expected behavior?

Update value in char-counter correctly

- AngularJS: 1.7.2
- AngularJS Material: 1.1.10 (all versions)
- OS: MacOS, Windows
- Browsers: All

**Comment by Splaktar** [comment]
I'm not sure what you mean by "NOT works". It is important to use our full issue template to avoid this kind of confusion.

I've updated the [CodePen](https://codepen.io/Splaktar/pen/BVgoVr?editors=1010) to be a little more clear, but in both the updated CodePen and in your CodePen, I don't see a problem.

What is the behavior that you expected?

How does the observed behavior differ from this?

If I set `forceDisabled` to true, then it should trigger the disabling of `md-maxlength` by passing it `''``. This seems to be working as expected.
